openpype/hosts/maya/api/pipeline.py

```diff
@@ -426,6 +468,49 @@ def on_before_save():
     return lib.validate_fps()
 
 
+def after_file_open():
```

Comment:
Function name is unclear - best to rename to `check_lock_on_current_file`? Additionally it looks like this function isn't used on "after open" but "on save"?

Reply:
It is implemented in "on_open", which binds to the kAfterOpen callback

Reply:
That was actually my point that callback to `kAfterOpen` should have `on_open` callback rather then `check_lock_on_current_file`.